subjects/DPxAI/objects-around/README.md

```diff
@@ -0,0 +1,118 @@
+## Objects Around
+
+> Mindful AI mode
+
+### Context
+
+Think about all the objects around you: like your book or robot friend. Each of these objects has different properties that describe it. For example, a robot can have a type, weight, and operational status. In JavaScript, we use objects to group these related properties together, making it easy to manage and access this information.
+
+### AI-Powered Learning Techniques
+
+`Visualization Technique:`
+This type of prompt encourages the AI to explain a concept using diagrams or visual representations to illustrate concepts.
+
+Find the examples across the subject ;)
+
+### Concepts
+
+### JavaScript Objects
+
+Objects in JavaScript are fundamental data structures used to group related values together. They are like a bag of values.
+
+### Example
+
+First, let's look at different types of variables:
+
+```js
+let type = "Talkative";
+let weight = 120.5;
+let isOperational = true;
+```
+
+Now, we can group them into an object. Objects are values too, so let's assign one to a robot variable:
+
+```js
+let robot = {
+  type: "Talkative",
+  weight: 120.5,
+  isOperational: true,
+};
+console.log(robot); // This will display the object 'robot'
+```
+
+Here, the robot variable is declared, and its value type is an object.
+
+### Object Literal Syntax:
+
+Objects are defined using curly brackets {}.
+
+```js
+let emptyRobot = {}; // an empty object
+```
+
+### Properties
+
+Objects consist of properties, each having a key and a value:
+
+```js
+let robot = {
+  type: "Talkative", // 'type' is the key, 'Talkative' is the value
+  weight: 120.5,
+  isOperational: true,
+};
+```
+
+Each property is separated by a comma ,. It's good practice to add a trailing comma to every property, though it's not required for the last one.
```

Comment:
Maybe format `,` or add a column like `... by a comma: ,.`, or any other way you feel might clarify that "," is the symbol related to the word "comma" :sweat_smile: 

The sentence like this might be confusing.